client/src/components/forms/Login.js

```diff
@@ -41,12 +47,12 @@ function Login() {
           type="password"
           name="password"
           required
-          value={input.password}
           onChange={(e) =>
             setInput((state) => ({ ...state, password: e.target.value }))
           }
         />
       </label>
+      <span className="error-msg">{}</span>
       <button type="submit">Login</button>
       <Link to="/">
         <div className="back">Back</div>
```

Comment:
This isn't really going back, though. It's fine, I suppose, but it'd be great if it actually went back instead of `/`. React-router has a way.